autotest/utilities/test_gdal_grid.py

```diff
@@ -887,6 +887,52 @@ def test_gdal_grid_clipsrc():
     assert not (cs == 0 or cs == 4672), 'bad checksum'
     ds = None
 
+###############################################################################
+# Test -tr
+
+
+def test_gdal_grid_tr():
+    if gdal_grid is None:
+        pytest.skip()
+
+    #################
+    outfiles.append('tmp/grid_count_70_70.tif')
+    try:
+        os.remove(outfiles[-1])
+    except OSError:
+        pass
+
+    # Create a GDAL dataset from the values of "grid.csv".
+    gdaltest.runexternal(gdal_grid + ' -txe 440720.0 441920.0 -tye 3751320.0 3750120.0 -tr 60 60 -ot Byte -l grid -a count:radius1=70.0:radius2=70.0:angle=0.0:min_points=0:nodata=0.0 data/grid.vrt ' + outfiles[-1])
+
+    # We should get the same values as in "ref_data/grid_count_70_70.tif"
```

Comment:
this file has not been committed

Reply:
> this file has not been committed

This isn't a new file. It was already (and is still) being used by another test:
https://github.com/OSGeo/gdal/blob/d7bbdeff5c705c3eac51e1f8500d09e467a27e00/autotest/utilities/test_gdal_grid.py#L635-L680

Thus, it [was already there before this PR]( https://github.com/OSGeo/gdal/blob/d7bbdeff5c705c3eac51e1f8500d09e467a27e00/autotest/utilities/ref_data/grid_count_70_70.tif) and still is.

Reply:
ah, ok my bad, I wrongly assumed it was a new one.